GTG/gtk/editor/editor.py

```diff
@@ -30,11 +30,12 @@
 from GTG.core.plugins.api import PluginAPI
 from GTG.core.plugins.engine import PluginEngine
 from GTG.core.task import Task
-from GTG.core.translations import _, ngettext
+# from GTG.core.translations import _, ngettext
 from GTG.gtk.editor import GnomeConfig
 from GTG.gtk.editor.calendar import GTGCalendar
 from GTG.gtk.editor.taskview import TaskView
-from GTG.gtk.help import add_help_shortcut
+# from GTG.gtk.help import add_help_shortcut
```

Comment:
However this is the reason for these things being commented out:
Since some of the core functions using these things are commented out / not in the code probably anymore, tests will fail and that's not what we want

![screen shot 2015-07-01 at 11 16 06](https://cloud.githubusercontent.com/assets/2866323/8451302/961ab20e-1fe2-11e5-9841-be0f6ba5b5fa.png)